lib/copy_peste/command/list_result.rb

```diff
@@ -0,0 +1,60 @@
+module CopyPeste
+  class Command
+    module ListResult
+
+      # Method used to display result list of module.
+      #
+      # @param result [Collection] collection data of result list
+      # @param name [String] Name of module, could be nil
+      def display_results(results, name)
+        results = (name && !name.empty?) ? results.where(:module_name => name.upcase) : results
```

Comment:
Why use #upcase here? Shouldn't the 'name' already match what is requested?
This method doesn't handle cases such as 'some_composed_word'.